megamek/src/megamek/client/ui/swing/RandomArmyDialog.java

```diff
@@ -784,34 +790,51 @@ public void actionPerformed(ActionEvent ev) {
         }
     }
 
-    public void windowActivated(WindowEvent arg0) {
-        //ignored
-    }
-
-    public void windowClosed(WindowEvent arg0) {
-        //ignored
-    }
-
-    public void windowClosing(WindowEvent arg0) {
-        setVisible(false);
-    }
-
-    public void windowDeactivated(WindowEvent arg0) {
-        //ignored
-    }
-
-    public void windowDeiconified(WindowEvent arg0) {
-        //ignored
-    }
-
-    public void windowIconified(WindowEvent arg0) {
-        //ignored
-    }
+//    public void windowActivated(WindowEvent arg0) {
```

Comment:
If we don't need these, how about deleting them?

Reply:
done.